src/webui/api/torrentscontroller.cpp

```diff
@@ -1362,18 +1362,22 @@ void TorrentsController::setDownloadLimitAction()
 
 void TorrentsController::setShareLimitsAction()
 {
-    requireParams({u"hashes"_s, u"ratioLimit"_s, u"seedingTimeLimit"_s, u"inactiveSeedingTimeLimit"_s});
+    requireParams({u"hashes"_s, u"ratioLimit"_s, u"seedingTimeLimit"_s, u"inactiveSeedingTimeLimit"_s, u"shareLimitAction"_s});
```

Comment:
Why not keep it optional?

Reply:
yeah just for consistency. 

though this will be a breaking change for existing users who just uses the webapi since the `/api/v2/torrents/setShareLimits` now requires `shareLimitAction` in the post data or else it will return an http 400